pest/src/position.rs

```diff
@@ -282,6 +282,25 @@ impl<'i> Position<'i> {
         false
     }
 
+    /// Skips until the one of the given `strings` is found. If none of the `strings` can be found,
+    /// this function will return `false` and its `pos` will not be updated.
+    #[inline]
+    pub(crate) fn skip_until_any(&mut self, strings: &[&str]) -> bool {
```

Comment:
We should probably only keep `skip_until_any` and remove `skip_until` with its depedency.

Reply:
I think I'll just have the latter delegate to the former as an API convenience